rsconnect/actions.py

```diff
@@ -538,6 +540,10 @@ def write_quarto_manifest_json(
     :param extra_files: Any extra files to include in the manifest.
     :param excludes: A sequence of glob patterns to exclude when enumerating files to bundle.
     :param image: the optional docker image to be specified for off-host execution. Default = None.
+    :param env_management_py: False indicates that the user is responsible for Python package installation
```

Comment:
```suggestion
    :param env_management_py: False prevents Connect from managing the Python environment for this bundle. The server administrator is responsible for installing packages in runtime environment on the server.
```

Something like that maybe? I wanna get away from saying it's the *user's* responsibility, because users likely won't have permission to do it.